src/data/markdown/docs/30 k6-browser/01 Get started/02 Running browser tests.md

```diff
@@ -0,0 +1,237 @@
+
+- testing real user flows on the frontend while generating a higher load in the backend
+- measuring backend and frontend performance in the same test execution
+- increased collaboration between backend and frontend teams since the same tool can be used
+
+To run a browser-level and protocol-level test concurrently, you can use [scenarios](/using-k6/scenarios/). 
+
+<Blockquote mod="note" title="">
+
+  Keep in mind that there is an additional performance overhead when it comes to spinning up a browser VU and that the resource usage will depend on the system under test.
+
+  </Blockquote>
+
+<CodeGroup labels={["script.js"]} lineNumbers={[true]}>
+
+```javascript
+import { chromium } from 'k6/experimental/browser';
+import { check } from 'k6';
+import http from 'k6/http';
+
+export const options = {
+  scenarios: {
+    browser: {
+      executor: 'constant-vus',
+      exec: 'browser',
+      vus: 1,
+      duration: '10s',
+    },
+    news: {
+      executor: 'constant-vus',
+      exec: 'news',
+      vus: 20,
+      duration: '1m',
+    },
+  },
+};
+
+export async function browser() {
+  const browser = chromium.launch({ headless: false });
+  const page = browser.newPage();
+
+  try {
+    await page.goto('https://test.k6.io/browser.php', { waitUntil: 'networkidle' });
+
+    page.locator('#checkbox1').check();
+
+    check(page, {
+      'checkbox is checked': page.locator('#checkbox-info-display').textContent() === 'Thanks for checking the box',
+    });
+  } finally {
+    page.close();
+    browser.close();
+  }
+}
+
+export function news() {
+  const res = http.get('https://test.k6.io/news.php');
+
+  check(res, {
+    'status is 200': (r) => r.status === 200,
+  });
+}
+```
+
+</CodeGroup>
+
+The preceding code contains two scenarios. One for the browser-level test called `browser` and one for the protocol-level test called `news`. Both scenarios are using the [constant-vus executor](/using-k6/scenarios/executors/constant-vus/) which introduces a constant number of virtual users to execute as many iterations as possible for a specified amount of time. 
+
+Since it's all in one script, this allows for greater collaboration amongst teams.
+
+## Run browser tests in a Docker container
+
+If you prefer working with Docker, you can run your browser test scripts in a Docker container using Docker Compose by creating a [Dockerfile](https://github.com/grafana/xk6-browser/blob/main/Dockerfile) and [docker-compose](https://github.com/grafana/xk6-browser/blob/main/docker-compose.yaml) file.
+
+To run the test, use the following command and replace `script.js` with your file.
+
+<CodeGroup labels={["CLI"]}>
+
+```bash
+docker-compose run -T K6_BROWSER_ENABLED=1 k6 run - <script.js
+```
```

Comment:
If this documentation is going to be merged after the release of v.0.43.0, should this example use the k6 docker image directly (considering that xk6-browser will be included in it) instead of referencing Docker and docker compose files that build a "custom" image through xk6?